docs/hub/datasets-image.md

```diff
@@ -0,0 +1,148 @@
+# Image Dataset
+
+This guide will show you how to configure your dataset repository with image files.
+
+A dataset with a supported structure and [file formats](./datasets-adding#file-formats) automatically has a Dataset Viewer on its dataset page on the Hub. Any additional information about your dataset - such as captioning or object detection - is automatically loaded as long as you include this information in a metadata file (`metadata.csv`/`metadata.jsonl`).
+
+## Only images
+
+If your dataset only consists of one column with images, you can simply store your image files at the root:
+
+```
+.
```

Comment:
for consistency with https://huggingface.co/docs/hub/datasets-manual-configuration
```suggestion
my_dataset_repository/
```